firmware_io/simulink_input.csv

```diff
@@ -1,2 +1,51 @@
-Variable Name, Datatype, Minimum Value, Maximum Value, Name Description, Value Description,
-DI_V_SteeringAngle, float,-1,1, Steering wheel angle," ""-1 is full left turn"," +1 is full right turn. THIS IS JUST AN EXAMPLE"""
+Variable Name,Datatype,Unit,Minimum Value,Maximum Value,Name Description,Value Description,
+DI_SteeringAngle,float,%,0,1,Steering wheel angle,0 is full left turn,1 is full right turn
```

Comment:
can you change the unit to "fraction", percent implies a range of 0-100
repeat for the other %, if applicable